MonoGame.Framework/Content/ContentReaders/VertexBufferReader.cs

```diff
@@ -34,11 +11,15 @@ class VertexBufferReader : ContentTypeReader<VertexBuffer>
     {
         protected internal override VertexBuffer Read(ContentReader input, VertexBuffer existingInstance)
         {
+			var buffer = existingInstance;
+
             var declaration = input.ReadRawObject<VertexDeclaration>();
-            var vertexCount = (int)input.ReadUInt32();
+            var vertexCount = input.ReadInt32();
```

Comment:
The field is a UInt32 (XNB file format document was available on MSDN but disappeared when create.msdn.com was retired), so it should be read as such.  That VertexBuffer takes an int for the vertexCount parameter is an inconsistency, but that is what XNA compatibility is all about.